### PR TITLE
Bugfix in scouting PV producer

### PR DESCRIPTION
**Pull request by swagata87**

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
type bug-fix

File: HLTrigger/JetMET/plugins/HLTScoutingPrimaryVertexProducer.cc [hunk 1]
```diff
@@ -71,6 +71,8 @@ void
 HLTScoutingPrimaryVertexProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
   desc.add<edm::InputTag>("vertexCollection", edm::InputTag("hltPixelVertices"));
+  descriptions.add("hltScoutingPrimaryVertexproducer", desc);
```
Comment (Martin-Grunewald) [inline HLTrigger/JetMET/plugins/HLTScoutingPrimaryVertexProducer.cc]: Capitalisation: ```"hltScoutingPrimaryVertexProducer"```

Comment (swagata87) [inline HLTrigger/JetMET/plugins/HLTScoutingPrimaryVertexProducer.cc]: OK, so I will make this small change and push in the same branch again.

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Pull request merged**